src/server/game/Entities/Creature/CreatureTemplate.h

```diff
@@ -0,0 +1,175 @@
+/*
+* Copyright (C) 2008-2018 TrinityCore <https://www.trinitycore.org/>
+* Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
+*
+* This program is free software; you can redistribute it and/or modify it
+* under the terms of the GNU General Public License as published by the
+* Free Software Foundation; either version 2 of the License, or (at your
+* option) any later version.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+* more details.
+*
+* You should have received a copy of the GNU General Public License along
+* with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef TRINITYCORE_CREATURETEMPLATE_H
+#define TRINITYCORE_CREATURETEMPLATE_H
+
+#include "DBCEnums.h"
+#include "Common.h"
+#include "SharedDefines.h"
+
+class WorldPacket;
+namespace WorldPackets {
+    namespace Query {
+        class QueryCreatureResponse;
```

Comment:
Please follow TC C++ codestyle, curly brackets get their own separate lines in core files:
```cpp
namespace WorldPackets
{
    namespace Query
    {
        class QueryCreatureResponse;
    }
}
```